compiler/src/dotty/tools/dotc/transform/sjs/PrepJSInterop.scala

```diff
@@ -981,7 +981,7 @@ class PrepJSInterop extends MacroTransform with IdentityDenotTransformer { thisP
 
       // Check that the resul type was explicitly specified
```

Comment:
drive-by typo suggestion:
```suggestion
      // Check that the result type was explicitly specified
```

Reply:
Thanks, fixed.